### PR TITLE
axiomatise a couple classes

### PR DESCRIPTION
**Pull request by justin2004**
Following up on:
https://github.com/CommonCoreOntology/CommonCoreOntologies/discussions/368

image

image

### CODE REVIEW

File: src/cco-modules/EventOntology.ttl [hunk 1]
```diff
@@ -587,6 +587,12 @@ cco:ActOfIssuingMassMediaPressRelease rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/ActOfLegalInstrumentUse
 cco:ActOfLegalInstrumentUse rdf:type owl:Class ;
                             rdfs:subClassOf cco:ActOfArtifactEmployment ;
+                            owl:equivalentClass [ owl:intersectionOf ( [ a owl:Restriction ;
```
Comment (neilotte) [inline src/cco-modules/EventOntology.ttl:590]: @justin2004 This axiom is too strong. It implies that any act performed by an agent that involves a legal instrument as a participant is an act of legal instrument use. The axiom does not imply that the agent actually needs to be using the participating legal instrument in the process, which seems to be required. From this follow counter examples: A scholar citing a legal instrument in an argument would seem to satisfy these axioms but not involve an act of legal instrument use.

**Review by neilotte — CHANGES_REQUESTED** [review]
See comment on equivalence axiom for act of legal instrument use.

File: src/cco-modules/EventOntology.ttl (hunk 1 above)
Comment (justin2004) [reply src/cco-modules/EventOntology.ttl]: hey @neilotte , good catch.
i've addressed that here:
image

now a `cco:ActOfLegalInstrumentUse` is a subclass (more specialized) of an act with an agent and a LegalInstrument is a participant.

**Comment by mark-jensen** [comment]
Closing for now, but see the associated discussion. More to follow.

**Pull request closed without merging**